soccer/gameplay/tactics/advance_zone_midfielder.py

```diff
@@ -0,0 +1,174 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+import math
+
+import composite_behavior
+import skills.move
+import evaluation.ball
+import evaluation.passing_positioning
+import evaluation.passing
+import evaluation.shooting
+import functools
+import plays.offense.adaptive_formation
+
+
+#2 midfielder rely on the future location of their teammate to pass quickly
+class AdvanceZoneMidfielder(composite_behavior.CompositeBehavior):
+    # Weights for the general field positioning
+    FIELD_POS_WEIGHTS = (0.01, 3, 0.02)#AdaptiveFormation.FIELD_POS_WEIGHTS
```

Comment:
This is just the same as `AdaptiveFormation.FIELD_POS_WEIGHTS`, can't you copy it in here?

I'd rather have one source for constants and tweak them if needed (unless the number is completely different).